scala/pekkohttp/build.sbt

```diff
@@ -0,0 +1,11 @@
+name := "server"
+scalaVersion := "3.5.2"
+
+val PekkoVersion = "1.1.0"
```

Comment:
could you use the same version for PekkoVersion and PekkoHttpVersion ?

Reply:
It looks like it's not possible => https://pekko.apache.org/docs/pekko-http/current/introduction.html#using-apache-pekko-http

btw, it's the same with Akka and Akka HTTP.

Reply:
maybe we should ping any core contributor for this, it appears not clear in the docs (at least for me), since version of both are pined

Reply:
Trust me, I know and have been used Akka. Akka is the actor framework. Later on they built Akka HTTP which is the HTTP framework on-top-of Akka. So Akka HTTP uses _Akka_ so it's quite natural that versions are different.